webview/src/experiments/components/table/styles.module.scss

```diff
@@ -338,10 +337,6 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
         background-color: $row-hover-background-color;
       }
 
-      .indicatorCount {
-        border-color: $row-hover-background-color;
```

Comment:
@maxagin, I took off the border of the indicators since VSCode doesn't have one for their indicators, but if we want to keep that effect, I don't see an issue with adding it back :) 

Figma:
<img width="338" alt="image" src="https://user-images.githubusercontent.com/43496356/189407889-7a873e58-9737-4b8b-9937-7500591af134.png">